lib/new-admin/graphql/schema.js

```diff
@@ -260,40 +290,39 @@ const typeDefs = gql`
   }
 
   type Query {
-    countries: [Country]
```

Comment:
You'll probably need to update this with the new fields

Reply:
Not sure if I understood